index.html

```diff
@@ -69,5 +69,181 @@ <h1>Hey There. I’m Ronald.</h1>
         </li>
       </ul>
     </main>
+    <!-- works section -->
+    <section class="works">
+      <h2 class="recent">My Recent Works</h2>
+
+      <div class="card">
+        <div class="projimage">
+          <img src="./images/projects/Img Placeholder.svg" alt="" />
+        </div>
+        <h2 class="cardheader">Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="languages">
+          <li>CSS</li>
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button>See project</button>
+      </div>
+      <div class="card2">
+        <div class="textitems">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
```

Comment:
- [ ] Nice Job enhancing the paragraph section. As you can see, the first header in the `h2` is still short of the template font of which yours is bolder and not extended . Kindly adjust it to fit the template space, which gives it a better feel.

## Figma:
![image](https://github.com/mj-e-boc/M-J-Portfolio/assets/50990925/3644b3db-09b3-4847-8b68-ac8e5e7beb30)

## Your design:
![image](https://github.com/mj-e-boc/M-J-Portfolio/assets/50990925/9d707d26-6af7-4127-ad63-4f281d68e32d)